docs/unified/hardkernel/initial-deployment.md

```diff
@@ -0,0 +1,36 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer. This document describes the process of building,
+installing and running flashrom on Ubuntu 24.04.
```

Comment:
It should also be possible to use DTS (with DPP subscription) when https://github.com/Dasharo/dts-scripts/pull/31 and https://github.com/Dasharo/meta-dts/pull/168 are be merged. Just tested to make sure it works.

Reply:
Does this look good to you?
https://github.com/Dasharo/docs/pull/903/commits/aa5f0b9e5099165a0cee2694371037c072af5492